addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -189,6 +188,8 @@ function getMatchedCSSRules(a) {
  *                           converted to images
  */
 function fontToImg($editable) {
+    var fonts = odoo.__DEBUG__.services["wysiwyg.fonts"];
```

Comment:
There is no mistake, it's because of our module system and the current structure. I'm going to talk to you about it, you never know you're going to find anything else.